loops/csharp/code.cs

```diff
@@ -0,0 +1,10 @@
+﻿var u = int.Parse(args[0]); // Get an input number from the command line
+var r = new Random().Next(10000); // Get a random number 0 <= r < 10k
```

Comment:
Use 'Random.Shared' please

Reply:
Doubt it will make much of a difference performance-wise, Random.Shared still needs to be allocated and initialized when first accessed. And since we're using the random object only once the initialization cost should be the same

Reply:
Actually, you were onto something, Random.Shared uses a private Random.ThreadSafeRandom class. I wrote this quick benchmark to check the instantiation performance.

`using System.Linq.Expressions;
using BenchmarkDotNet.Attributes;
using BindingFlags = System.Reflection.BindingFlags;

namespace Benchmarks;

[MemoryDiagnoser]
public class RandomCreationBenchmarks
{
    private static Func<Random> ThreadSafeConstructorCall;

    [GlobalSetup]
    public void Setup()
    {
        var threadSafeRandomClass = typeof(Random).GetNestedType("ThreadSafeRandom", System.Reflection.BindingFlags.NonPublic)!;
        var ctorCall = Expression.New(threadSafeRandomClass.GetConstructor(BindingFlags.Public | BindingFlags.NonPublic | BindingFlags.Instance, Type.EmptyTypes));
        ThreadSafeConstructorCall = Expression.Lambda<Func<Random>>(ctorCall).Compile();
    }
    
    [Benchmark]
    public Random CreateParameterlessRandom() => new Random();

    [Benchmark]
    public Random CreateThreadsafeRandom() => ThreadSafeConstructorCall();
}`

Results on my rig are:

![image](https://github.com/user-attachments/assets/390aeba9-4002-4840-aee9-85291bd0c3df)

So Random.Shared does seem to be a lot faster when it comes to instantiation

Reply:
Accidentally ran it using .net SDK 6, here's the correct results

![image](https://github.com/user-attachments/assets/e9a360c3-9fe2-4088-be15-d2ee7b4e5465)

Reply:
Accidentally ran the benchmarks with .net 6, here are the correct results

![image](https://github.com/user-attachments/assets/6a9f49b2-f99e-4f14-86b1-118b9a4e63af)